cellmlmanip/model.py

```diff
@@ -555,21 +555,57 @@ def get_symbol_by_ontology_term(self, namespace_uri, local_name):
             raise ValueError('Multiple variables annotated with {%s}%s' %
                              (namespace_uri, local_name))
 
-    def _get_symbols_by_rdf(self, predicate, object_=None):
+    def get_rdf_annotation(self, predicate):
+        """Searches the RDF graph for variables annotated with the given predicate
```

Comment:
Except this doesn't just find variables, judging by the example given!

Also, how do you distinguish in the results what has been found as the subject, between variables and non-variables? I think you probably know generally whether you're hunting for model annotations or variable annotations; in the former case you want to specify the subject in your query for efficiency, and in the latter you want to ensure you only get variables.

Reply:
That's correct it doesn't just find variables, will update the docstring. It's specifically for model annotations. But it could be used for any rdf. There is absolutly no link back to model variables, if you want that use get_symbols_by_rdf.

   <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
      <rdf:Description rdf:about="#test_simple_odes">
          <named-attribute xmlns="https://chaste.comlab.ox.ac.uk/cellml/ns/pycml#">
             <rdf:Description>
                <name rdf:datatype="http://www.w3.org/2000/10/XMLSchema#string">SuggestedForwardEulerTimestep</name>
                <value rdf:datatype="http://www.w3.org/2000/10/XMLSchema#double">0.0002</value>
             </rdf:Description>
          </named-attribute>
      </rdf:Description>
   </rdf:RDF>